### ISSUE TITLE
Vulnerability Scanner does not detect initial packages for manager nodes

### ISSUE THREAD

**Issue by Rebits** [wazuh/wazuh#22859]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.8.0-beta5 | Vulnerability Scanenr | Manager | Packages | Ubuntu22 |

### Description 

It has been detected in [Release 4.8.0 - Beta 5 - E2E UX tests - Vulnerability Detection](https://github.com/wazuh/wazuh/issues/22812), that no vulnerabilities are detected for manager node. (Check https://github.com/wazuh/wazuh/issues/22812#issuecomment-2042466267):
```
root@ip-172-31-87-231:/home/ubuntu# curl -k -u admin:PASSWORD https://172.31.87.231:9200/wazuh-states-vulnerabilities/_search -H 'Content-Type: application/json' -d '{
  "size": 1000,
  "query": {

    "bool": {
      "must": [
        {
          "match": {
            "agent.id": "000"
          }
        }
      ]
    }
  }
}'
{"took":0,"timed_out":false,"_shards":{"total":1,"successful":1,"skipped":0,"failed":0},"hits":{"total":{"value":0,"relation":"eq"},"max_score":null,"hits":[]}}root@ip-172-31-87-231:/home/ubuntu# 
```

 
Regarding default internal options manager scan should be enabled by default
```
# Vulnerability detector - Enable or disable the scan manager 
# 0. Enabled 
# 1. Disabled 
vulnerability-detection.disable_scan_manager=0
```

Only new packages will be scanned. 

For this case, we have installed node and axios packages. However, already vulnerable packages present in the manager were ignored.

![manager](https://github.com/wazuh/wazuh/assets/11089305/3b9a0919-31a4-4956-8886-4e724a00b8ae)

### Steps to reproduce
- Deploy an AIO
- Wait until feeds are downloaded
- Check that no vulnerabilities are found for the manager

### Evidences

**Manager Vulnerabilities**: [vulnerabilities_inventory-2024-04-09T17 20 16.743Z.csv](https://github.com/wazuh/wazuh/files/14921954/vulnerabilities_inventory-2024-04-09T17.20.16.743Z.csv)
- **Manager Logs**:  [manager.zip](https://github.com/wazuh/wazuh/files/14922236/manager.zip)

**Comment by GabrielEValenzuela** [comment]
On it! I'll back ASAP with the analysis

**Comment by GabrielEValenzuela** [comment]
# Analysis

## Steps

- Clone branch 4.8.0
- Install manager
- Download content
- Check installed version
```bash
root@manager480:/var/ossec# ./bin/wazuh-control info
WAZUH_VERSION="v4.8.0"
WAZUH_REVISION="40807"
WAZUH_TYPE="server
```

## Result
All packages installed in the manager have been scanned successfully 🟢

**Comment by MARCOSD4** [comment]
During the testing process performed in https://github.com/wazuh/wazuh/issues/22828, we have been able to confirm this behavior, but, after investigating what was happening, the problem was related to https://github.com/wazuh/wazuh/issues/22871 because, once the agents were registered, the vulnerability detector module was enabled and therefore, as indicated there, no vulnerability of the initial packages was being generated for either the agents or the managers. That is, once the environment is configured in this way, the problem reported in this issue is produced, no vulnerabilities are generated for the initial packages of the manager nodes. The environment in which I managed to reproduce it is [this one](https://ci.wazuh.info/job/Wazuh_QA_environment/1072/), with one master, one worker, three agents connect to the master and the other two connect to the worker.
